.github/workflows/build_image.yml

```diff
@@ -3,6 +3,7 @@ name: Docker Image
 on:
   push:
     branches: [ "master" ]
+  workflow_dispatch:
```

Comment:
is this a valid yaml?

Reply:
it should be, worst case I'll make another PR

Reply:
I think it's not

Reply:
Yaml validator passes. I'll see if I can test the action itself before merging

Reply:
https://github.com/traveltime-dev/web-dev/blob/fcadea8fad62dc5d8a0e4470fdbe4a46fa565800/.github/workflows/demo-apps-update-dev.yaml#L4 same thing is done here